pkg/operator/status.go

```diff
@@ -289,9 +291,99 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if updating and degraded
+	if !updating && !degraded {
+		if supported, v := optr.isKubeletSkewSupported(); !supported {
+			coStatus.Reason = "KubeletSkewUnsupported"
+			coStatus.Message = fmt.Sprintf("machine config pool contains nodes with an unsupported kubelet version skew. Please see `oc get nodes` for details and upgrade all nodes so that they have a kubelet Version of at least %v.", getMinimalSkewSupportNodeVersion(v))
+		}
```

Comment:
can we also add an event here?

see an example here: https://github.com/openshift/machine-config-operator/blob/4f555d05550bca9a67f5354afa2721cea5e36b02/pkg/operator/status.go#L44

Reply:
@kikisdeliveryservice I added the event. PTAL.

Reply:
Shouldn't this event be of type "EventTypeWarning" ?